### DISCUSSION TITLE
Dependencies

### DISCUSSION THREAD

**Discussion by nickkrisulevicz** [ctavolazzi/NovaSystem#1] [Ideas]
Hello!

I came across this repo when thecoffeejesus linked it in a reddit comment. I was very interested in using it to do some problem solving and to see how capable the project is. The one suggestion I might add is to list each of the dependencies in the readme, so that everyone may install them so the project will work. I am very interested to see how capable Nova will be!

Thanks,
Nick

**Comment by nickkrisulevicz** [comment]
UPDATE: I read a later comment saying to simply paste the prompt into ChatGPT, forget what I said before!

**Reply by ctavolazzi** [reply]
Hello! I'm the developer

Thanks for your idea, I really appreciate it. The whole project file was a mess and I've been spending all day cleaning it up.

Over the weekend I'm going to be uploading test files for the Nova CLI, and then I'll be working on the Obsidian plugin Winfo, which runs on the NovaSystem. 

Winfo is a sister project that will allow documents to dynamically update one another inside an Obsidian vault.

I hope you come back here in the future! I've been working on this for a long time. Cheers!
